bin/debug/upload-results.ts

```diff
@@ -43,7 +63,7 @@ class FileSender extends DebugRequest {
 
 const sender = new FileSender()
 sender.perform().then(() => {
-    console.log(
+    console.info(
         'Results must be decrypted using ONLY tests/support/private_key.pem.  normal reviewer keys will not work.  use:\ncat tests/support/private_key.pem | pbcopy',
     )
 })
```

Comment:
@chrisbendel this file lets you easily test only the mgmt side of things, documented in README as:
```bash
npx tsx bin/debug/upload-results.ts  -j <uuid of job> -r <path to file to upload as results> -l <path of file to upload as logs>
```